src/Persistence/Initialization/Version095d/ChaosMixes.cs

```diff
@@ -36,26 +38,24 @@ public override void Initialize()
         chaosGoblin.ItemCraftings.Add(this.ChaosWeaponCrafting());
         chaosGoblin.ItemCraftings.Add(this.DinorantCrafting());
 
-        chaosGoblin.ItemCraftings.Add(this.ItemLevelUpgradeCrafting(3, 10, 1_000_000));
-        chaosGoblin.ItemCraftings.Add(this.ItemLevelUpgradeCrafting(4, 11, 2_000_000));
+        chaosGoblin.ItemCraftings.Add(this.ItemLevelUpgradeCrafting(3, 10));
+        chaosGoblin.ItemCraftings.Add(this.ItemLevelUpgradeCrafting(4, 11));
         chaosGoblin.ItemCraftings.Add(this.FirstWingsCrafting());
 
         chaosGoblin.ItemCraftings.Add(this.DevilSquareTicketCrafting());
     }
 
-    private ItemCrafting ItemLevelUpgradeCrafting(byte craftingNumber, byte targetLevel, int money)
+    private ItemCrafting ItemLevelUpgradeCrafting(byte craftingNumber, byte targetLevel)
     {
         var crafting = this.Context.CreateNew<ItemCrafting>();
         crafting.Name = $"+{targetLevel} Item Combination";
         crafting.Number = craftingNumber;
         var craftingSettings = this.Context.CreateNew<SimpleCraftingSettings>();
 
         crafting.SimpleCraftingSettings = craftingSettings;
-        craftingSettings.Money = money;
-        craftingSettings.SuccessPercent = (byte)(60 - (((targetLevel - 10) / 2) * 5));
-        craftingSettings.SuccessPercentageAdditionForExcellentItem = -10;
-        craftingSettings.SuccessPercentageAdditionForAncientItem = -15;
-        craftingSettings.SuccessPercentageAdditionForSocketItem = -20;
+        craftingSettings.Money = 2_000_000 * (targetLevel - 9);
+        craftingSettings.SuccessPercent = (byte)(targetLevel == 10 ? 50 : 45);
+        craftingSettings.SuccessPercentageAdditionForLuck = 25;
```

Comment:
Another troubling issue is CM rates. These appeared to have changed between versions. But until here it was pretty simple with only 2 values and the additional for luck.
![image](https://github.com/user-attachments/assets/819c3999-ec06-4bc0-ad6b-06382b71180b)